qemu/tests/blk_commit.py

```diff
@@ -0,0 +1,65 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_misc
+
+from avocado.core import exceptions
+
+from qemu.tests import block_copy
+
+
+class BlockCommit(block_copy.BlockCopy):
+
+    """
+    base class for block commit tests
+    """
+
+    def __init__(self, test, params, env, tag):
+        super(BlockCommit, self).__init__(test, params, env, tag)
+
+    @error_context.context_aware
+    def start(self):
+        """
+        start block device committing job;
+        """
```

Comment:
skip test this feature with hmp, it monitor is not qmp please raise TestError directly